modules/api/pom.xml

```diff
@@ -62,5 +62,11 @@
             <artifactId>junit-jupiter-engine</artifactId>
             <scope>test</scope>
         </dependency>
+      <dependency>
+        <groupId>org.eclipse.xpanse.modules</groupId>
+        <artifactId>service</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
```

Comment:
we can use the project.version instead of fixed version

Reply:
Yes. I will replace the fixed version.